lib/money/money.rb

```diff
@@ -138,11 +138,11 @@ def as_json(*args)
   end
 
   def to_xml(options = {})
-    to_s
-  end
-
-  def as_xml(*args)
-    to_s
+    if options.try(:[], :builder)
+      options[:builder].tag!(options[:root], to_s, {:type=>"decimal"})
```

Comment:
`options[:root]` is required here (unless you want to provide a default, but I can't think of a sane way to do that). So, can you use `options.fetch(:root)` which will raise a `KeyError` if it isn't provided.